app/models/miq_request_workflow.rb

```diff
@@ -192,7 +192,17 @@ def validate(values)
         if fld[:required] == true
           # If :required_method is defined let it determine if the field is value
           unless fld[:required_method].nil?
-            fld[:error] = send(fld[:required_method], f, values, dlg, fld, value)
+            # Check to see if we have a list of validation methods
+            if fld[:required_method].kind_of? Enumerable
+              fld[:required_method].each do |method|
+                fld[:error] = send(method, f, values, dlg, fld, value)
+                # Bail out early if we see an error
+                break unless fld[:error].nil?
+              end
+            else
+              fld[:error] = send(fld[:required_method], f, values, dlg, fld, value)
+            end
+
```

Comment:
So you had to do this because nothing else has two validation methods? @gmcculloug would this be helpful for other dialogs as well?
Alternatively could you have a single gce specific `:validate_vm_name` that runs the current checks plus the regex check?